### DISCUSSION TITLE
a Single Use Linux?

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#425] [General]
Can a spin of Spiral Linux be made to boot it - run a few tasks, access secure encrypted data (maybe even bitlocked data) and after a reboot it all just goes away?  Of course each new boot the system is fresh and some aspects will likely need to be set up again but in the long run this type of enviroment will be less likely a hackable surface and generally more secure plus it is mobile and less tied to any one laptop or system...   So, I am throwing this out there.   The internet has typically been unforgiving with new or kookie ideas so please try not to tear me into too many shreds  :P 

Thx/Bill

**Comment by geckolinux** [comment]
Hi there Bill, that sort of a use case definitely would make sense in certain scenarios, but it's not a focus of SpiralLinux. With that said, you could definitely use the SpiralLinux live system that way, and as you say it all goes away after a reboot.

**Comment by manujchandra** [comment]
I think you are looking for [Tails](https://tails.net/index.en.html)

**Reply by ghost** [reply]
Tails is fine EXCEPT it doesn't allow always accessing Encrypted data ... prolly better to just use Spiral as I have no need for darkweb or other law avoiding things  :P  ... I am perfectly fine avoiding hackers and law enforcement on my own   lol :)

Sorry/Bill 
System:
  Kernel: 6.6.13+bpo-amd64 arch: x86_64 bits: 64 
Machine:
  Type: Desktop System: Apple product: MacPro5,1 v: 0.0 serial: <filter> Chassis: type: 7
    v: Mac-F221BEC8 serial: <filter>
  Mobo: Apple model: Mac-F221BEC8 serial: <filter> UEFI: Apple v: MP51.88Z.0089.B00.1806081708
    date: 06/08/18
CPU:
  Info: 2x 6-core model: Intel Xeon X5670 bits: 64 type: MT MCP SMP arch: Nehalem rev: 2 cache:
